examples/with-i18n-rosetta/lib/i18n.js

```diff
@@ -44,13 +26,25 @@ export default function I18n({ children, locale, lngDict }) {
       activeLocaleRef.current = l
       if (dict) {
         i18n.set(l, dict)
-        setActiveDict(dict)
-      } else {
-        setTick((tick) => tick + 1)
       }
+      // force rerender to update view
+      setTick((tick) => tick + 1)
```

Comment:
It doesn't work for me. I get an Unhandled Runtime Error on `setTick((tick) => tick + 1)` on loading:

Unhandled Runtime Error
Error: Maximum update depth exceeded. This can happen when a component repeatedly calls setState inside componentWillUpdate or componentDidUpdate. React limits the number of nested updates to prevent infinite loops.

Reply:
Fixed!